docs/sphinx/user_guide/feature/loop_basic.rst

```diff
@@ -344,17 +344,18 @@ the same team. The ``RAJA::launch`` interface has three main concepts:
     device execution environment, which enables run time selection of
     kernel execution.
 
-  * ``RAJA::LaunchParams`` type. This type takes a number of teams and and a
-    number of threads as arguments.
+  * ``RAJA::LaunchParams`` type. This type takes a number of teams, threads
+    per team, and optionally size of dynamic shared memory in bytes.
 
   * ``RAJA::loop`` template. These are used to define hierarchical
     parallel execution of a kernel. Operations within a loop are mapped to
     either teams or threads based on the execution policy template parameter
     provided.
 
-Team shared memory is available by using the ``RAJA_TEAM_SHARED`` macro. Team
-shared memory enables threads in a given team to share data. In practice,
-team policies are typically aliases for RAJA GPU block policies in the
+Team shared memory is available by using the ``RAJA_TEAM_SHARED`` macro on
+statically sized arrays or via dynamic allocation in the ``RAJA::LaunchParams``
+method. Team shared memory enables threads in a given team shared access to a data buffer. In
+practice, team policies are typically aliases for RAJA GPU block policies in the
```

Comment:
does "typically aliases for RAJA GPU policies" need an explanation? i'm not sure what it means. which GPU policies?

Reply:
Right, this is old idea I had for how to introduce launch... I've pushed up a reworded version that I think makes more sense